application/cloud_api.py

```diff
@@ -63,42 +115,239 @@ def uploadFiles(self, types, paths, callback):
-    local = CloudWizard('localhost')
-    #server = TrafficCloud_Server("/project/path/goes/ahere","localhost")
-    print remote.server_ip
-    print remote.readIds()
-    print local.server_ip
-    print local.readIds()
\ No newline at end of file
+    print "Syntax looks fine!"
+
+    ###########################################################################
+    # Setup CloudWizard
+    ###########################################################################
+    #remote = CloudWizard('10.7.88.26')
+    #remote = CloudWizard('10.26.89.15')
+    remote = CloudWizard('10.7.24.11')
+
+    ###########################################################################
+    # Upload Video
+    ###########################################################################
+    #video_path = '/home/user/Documents/output.mp4'
+    video_path = '/home/user/Documents/stmarc_video.avi'
+    #video_path = '/home/user/Documents/Harvey_30min_day.mp4'
+    id = remote.uploadVideo(video_path)['identifier']
+
+    ###########################################################################
+    # Upload Homography
+    ###########################################################################
+    print id
+    aerial = [
+            (695.7036743164062, 406.8148193359375),
+            (819.7777709960938, 240.1481475830078),
+            (856.8148193359375, 553.111083984375),
+            (830.888916015625, 390.1481628417969),
+            (932.74072265625, 397.5555419921875)]
+    camera = [
+            (614.2222290039062, 703.111083984375),
+            (936.4444580078125, 419.77777099609375),
+            (197.55555725097656, 630.888916015625),
+            (519.7777709960938, 330.8888854980469),
+            (558.6666870117188, 536.4444580078125)]
+    aerial_path = '/home/user/Documents/TrafficGUIs/project_dir/TestClassification/homography/aerial.png'
+    camera_path = '/home/user/Documents/TrafficGUIs/project_dir/TestClassification/homography/camera.png'
+    remote.uploadHomography(\
+            aerial_path, camera_path,\
+            id, 0.05, aerial, camera)
+
+
+    ###########################################################################
+    # Test Configs
+    ###########################################################################
+ 
+    remote.configFiles(id,\
+                max_features_per_frame = 1001,\
+                num_displacement_frames = 11,\
+                min_feature_displacement = 0.000099,\
+                max_iterations_to_persist = 201,\
+                min_feature_frames = 14,\
+                max_connection_distance = 0.99,\
+                max_segmentation_distance = 0.69)
+    remote.testConfig('object', id)
+    remote.testConfig('feature', id)
+
+    ###########################################################################
+    # Run Analysis Route
+    ###########################################################################
+    remote.analysis(id,'philip.seger@students.olin.edu')
```

Comment:
Definitely would remove this email address. No need to spam Philip with random messages from other people.